tests/Tests/200__monitor_and_manage/210__alerts/210__alerts.robot

```diff
@@ -0,0 +1,289 @@
+...                                     203__alerts/notebook-pvc-usage/fill-notebook-pvc-to-100.ipynb
+
+
+*** Test Cases ***
+Verify No Alerts Are Firing Except For DeadManSnitch    # robocop: disable:too-long-test-case
+    [Documentation]    Verifies that, in a regular situation, only the DeadManSnitch alert is firing
+    [Tags]    Smoke
+    ...       Tier1
+    ...       ODS-540
+
+    Verify Alert Is Firing And Continue On Failure
+    ...    DeadManSnitch    DeadManSnitch
+
+    Verify Alert Is Not Firing And Continue On Failure
+    ...    Builds    JupyterHub image builds are failing    alert-duration=120
+
+    Verify Alert Is Not Firing And Continue On Failure
+    ...    RHODS-PVC-Usage    User notebook pvc usage above 90%    alert-duration=120
+
+    Verify Alert Is Not Firing And Continue On Failure
+    ...    RHODS-PVC-Usage    User notebook pvc usage at 100%    alert-duration=120
+
+    Verify "RHODS Route Error Burn Rate" Alerts Are Not Firing And Continue On Failure
+
+    Verify "RHODS Probe Success Burn Rate" Alerts Are Not Firing And Continue On Failure
+
+Verify Alert RHODS-PVC-Usage-Above-90 Is Fired When User PVC Is Above 90 Percent
+    [Documentation]    Runs a jupyter notebook to fill the user PVC over 90% and
+    ...    verifies that alert "User notebook pvc usage above 90%" is fired
+    [Tags]    Tier2
+    ...       ODS-516
+
+    Fill Up User PVC    ${NOTEBOOK_REPO_URL}    ${TEST_ALERT_PVC90_NOTEBOOK_PATH}
+
+    Prometheus.Wait Until Alert Is Firing    ${RHODS_PROMETHEUS_URL}
+    ...    ${RHODS_PROMETHEUS_TOKEN}
+    ...    RHODS-PVC-Usage
+    ...    User notebook pvc usage above 90%
+    ...    alert-duration=120
+
+    [Teardown]    Teardown PVC Alert Test
+
+Verify Alert RHODS-PVC-Usage-At-100 Is Fired When User PVC Is At 100 Percent
+    [Documentation]    Runs a jupyter notebook to fill the user PVC over 100% and
+    ...    verifies that alert "User notebook pvc usage at 100%" is fired
+    [Tags]    Tier2
+    ...       ODS-517
+
+    Fill Up User PVC    ${NOTEBOOK_REPO_URL}    ${TEST_ALERT_PVC100_NOTEBOOK_PATH}
+
+    Prometheus.Wait Until Alert Is Firing    ${RHODS_PROMETHEUS_URL}
+    ...    ${RHODS_PROMETHEUS_TOKEN}
+    ...    RHODS-PVC-Usage
+    ...    User notebook pvc usage at 100%
+    ...    alert-duration=120
+
+    [Teardown]    Teardown PVC Alert Test
+
+Verify Alert "RHODS Route Error Burn Rate" Is Fired When Traefik Is Down    # robocop: disable:too-long-test-case
+    [Documentation]    Verifies that alert "RHODS Route Error Burn Rate" is fired when traefik-proxy is not working
+    [Tags]    Tier3
+    ...       ODS-738
+
+    Skip    msg=This alert was disabled in RHODS 1.3.0. More info at RHODS-2101
+
+    Skip Test If Alert Is Already Firing    ${RHODS_PROMETHEUS_URL}
+    ...    ${RHODS_PROMETHEUS_TOKEN}
+    ...    SLOs-haproxy_backend_http_responses_total
+    ...    RHODS Route Error Burn Rate
+    ...    alert-duration=120
```

Comment:
alert duration we should add unit of time to the number like _120s****_

Reply:
This value is not a Robot Framework timeout, it is matching a value returned by the prometheus API. I prefer to leave it like this